src/components/Comments.jsx

```diff
@@ -1,5 +1,41 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { getComments } from "../utils/api";
 
 export default function Comments() {
-  return <div>Comments</div>;
+  const [comments, setComments] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const { review_id } = useParams();
+
+  useEffect(() => {
+    setIsLoading(true);
+    setTimeout(() => {
+      getComments(review_id).then((commentsFromApi) => {
+        setComments(commentsFromApi);
+        setIsLoading(false);
+      });
+    }, 0);
+  }, [review_id]);

```

Comment:
A thought for later (don't need to change any of this now!): 

-this pattern of states you are using in the component, the useEffect, and setting loading is used in very similar ways in different components
-you could extract a lot of this logic out to be a generic custom hook

If you have time at the end of the week you could come back and implement this.  For now leave it as it is (it works well!)